src/main/settings.py

```diff
@@ -73,10 +73,13 @@
     }
 """
 
+USE_JWKS_TEST_KEY = os.getenv('USE_JWKS_TEST_KEY', 'false').lower() == 'true'
+PUB_JWKS = JWKS_TEST_KEY if not os.getenv('PUB_JWKS') and USE_JWKS_TEST_KEY else os.getenv('PUB_JWKS')
```

Comment:
Dit vind ik verwarrend, ik denk dat als `USE_JWKS_TEST_KEY = True`, hij altijd de testkey moet pakken, ongeacht of er een andere key in de env staat.